test/core/memory_usage/memory_usage_test.cc

```diff
@@ -25,15 +25,18 @@
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
 #include "absl/strings/str_cat.h"
+#include "util/logging.h"
 
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>
+#include <grpc/support/time.h>
 
 #include "src/core/lib/gpr/string.h"
 #include "src/core/lib/gprpp/host_port.h"
 #include "test/core/util/port.h"
 #include "test/core/util/subprocess.h"
+#include "test/core/util/test_config.h"
 
 ABSL_FLAG(std::string, benchmark_name, "call", "Which benchmark to run");
```

Comment:
We're not testing the `channel` branch in CI with this change.

Suggestion:
1. rename `benchmark_name` to `benchmark_names`, default it to `"call,channel"`
2. change main() to be:
```
int main(int argc, char**argv) {
  // code to ~line 103
  auto benchmarks = absl::StrSplit(absl::GetFlag(FLAGS_benchmark_name));
  for (const auto& benchmark : benchmarks) {
    int r = RunBenchmark(benchmark, ...); // ... is whatever context we need to pass down
    if (r != 0) return r;
  }
}
```
3. Write the new RunBenchmark function:
```
int RunBenchmark(absl::string_view benchmark, ...) {
  if (benchmark == "call") return RunCallBenchmark(...);
  else if (benchmark == "channel") return RunChannelBenchmark(...);
  else { log("that's not a benchmark"); return 1; }
}
```

Reply:
Would it be better to do this or to have the calling python script call the benchmark again but with benchmark_name="channel"? Since right now we only have one size flag since we weren't accounting for multiple benchmarks to be run at once. Also I'm not sure if making it run multiple benchmarks within the driver will cause issues for the python script when working with the output.

Reply:
What is required for the CI to test the channel branch?

Reply:
the `grpc_cc_test` will invoke the binary with no arguments - we can tweak it to add some arguments, but that will be clumsy.

Another way to approach this might be:
1. Split out the RunCallBenchmark, RunChannelBenchmark functions into a new library (or libraries)
2. have the main.cc file call these
3. have a call_test.cc, channel_test.cc program that just calls the appropriate benchmark driver

Reply:
I think I'll go with the first suggestion and just change size to sizes so that if benchmark_names="call, channel" and sizes =  "50000, 10000" then call will run with 50000 calls and channel will run with 10000 channels. Or perhaps it might be better to do benchmark="call_50000, channel_10000"?

Reply:
Discussed over call, going with first suggestion and changing default size to smaller. Different sizes not needed as the default is just for the CI to test both, and normally it will be called with only one benchmark.